demisto_sdk/commands/content_graph/objects/repository.py

```diff
@@ -11,10 +11,20 @@
 from demisto_sdk.commands.common.cpu_count import cpu_count
 from demisto_sdk.commands.common.logger import logger
 from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.repository import RepositoryParser
 
 USE_MULTIPROCESSING = False  # toggle this for better debugging
 
 
+def from_path():
```

Comment:
name is unclear, which path?
I think, either allow providing it or call it `all_content_repo()`